Geometry_Engine/Query/StartDir.cs

```diff
@@ -34,13 +35,19 @@ public static partial class Query
         /**** Public Methods - Curves                   ****/
         /***************************************************/
 
+        [Description("Gets the tangent vector at the start of the Arc.")]
+        [Input("arc", "The Arc from which to get the tangent at the start point.")]
+        [Output("startTan", "The tangent vector at the start of the Arc.")]
         public static Vector StartDir(this Arc arc)
         {
             return arc.CoordinateSystem.Y.Rotate(arc.StartAngle, arc.CoordinateSystem.Z);
         }
 
         /***************************************************/
 
+        [Description("Gets the tangent vector at the start of the Circle.")]
+        [Input("circle", "The Circle from which to get the tangent at the start point.")]
+        [Output("startTan", "The tangent vector at the start of the Circle.")]
```

Comment:
Shouldn't it be `startDir` to fit into method naming convention? Refers to all methods here, also end dir

Reply:
My thinking here was to use another name, to add information. Basically use a synonym as it could be called either (and we could have called the methods StartTan and EndTan instead really).

Can change if you disagree though.

Reply:
I think I would try being consistent, also not sure how many users would be confused by tangent, while direction is very obvious even if less precise. But no strong opinion here - I am just a guest in this toolkit so I do not have the right to push conventions 😉